c/datasetjson.c

```diff
@@ -110,6 +118,16 @@ int streamDataset(Socket *socket, char *filename, int recordLength, jsonPrinter
       }
     }
     fclose(in);
+    rcEtag = icsfDigestFinish(&digest, hash);
+
+    // Convert hash text to hex.
+    char eTag[70] = "";
+    memset(eTag, '\0', strlen(eTag));
+    sprintf(eTag, "%6s", "eTag->");
```

Comment:
I haven't tested [simpleHexPrint](
https://github.com/zowe/zowe-common-c/blob/staging/c/utils.c#L549), but it seems promising. The code could something like this:
```c
simpleHexPrint(&eTag[sizeof(ETAG_TEXT) - 1], hash, digest.hashLength);
```